cognite/neat/graph/loaders/core/models.py

```diff
@@ -67,7 +67,7 @@ def preprocess_fields(cls, values: dict):
                 msg += f"has multiple values {values[field]}, "
                 msg += f"these values will be joined in a single string: {', '.join(values[field])}"
                 logging.info(msg)
-                values[field] = ", ".join(values[field])[: METADATA_VALUE_MAX_LENGTH - 1]
+                values[field] = ", ".join(sorted(values[field]))[: METADATA_VALUE_MAX_LENGTH - 1]
```

Comment:
Need deterministic behavior of the current loader to be able to compare. Since the query is not sorted these values can come in an arbitrary order.